deploy/sre-prometheus/100-cluster-version-operator.PrometheusRule.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: sre-cluster-version-operator
+    role: alert-rules
+  name: sre-cluster-version-operator
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-cluster-version-operator
+    rules:
+    - alert: ClusterOperatorDegradedSRE
+      annotations:
+        summary: Cluster operator has been degraded for 1 day.
+        description: The {{ "{{ $labels.name }}" }} operator is degraded because {{ "{{ $labels.reason }}" }}, and the components it manages may have reduced quality of service.  Cluster upgrades may not complete. For more information refer to 'oc get -o yaml clusteroperator {{ "{{ $labels.name }}" }}'{{ "{{ with $console_url := \"console_url\" | query }}{{ if ne (len (label \"url\" (first $console_url ) ) ) 0}} or {{ label \"url\" (first $console_url ) }}/settings/cluster/{{ end }}{{ end }}" }}.
+      expr: |
+        max by (namespace, name, reason)
+        (
+          (
+            cluster_operator_conditions{job="cluster-version-operator", condition="Degraded", name =~ "kube-apiserver|kube-controller-manager|kube-scheduler"}
+            or on (namespace, name)
+            group by (namespace, name) (cluster_operator_up{job="cluster-version-operator", name =~ "kube-apiserver|kube-controller-manager|kube-scheduler"})
+          ) == 1
+        )
+      for: 1d
+      labels:
+        link: "https://github.com/openshift/ops-sop/blob/master/v4/alerts/ClusterOperatorDegraded.md"
+        namespace: openshift-monitoring
+        needs_attention: "true"
```

Comment:
What does `needs_attention: true` mean in practice?

Reply:
In practice, it means that this is a warning alert, that should not be deleted and should raise an incident on PD.
This has to be used for alerts that we delete but that we want to look at after some time without considering them as critical, as it will put the cluster in danger in the future...
The none deletion, will be handled by https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/90712, and the documentation by https://github.com/openshift/ops-sop/pull/2947

Reply:
Thanks for the info. I'm not really seeing the value though - why not just make it critical?

I don't know if we want Primary having to make decisions on how important a page is. Either we should respond to it ASAP or it's something that we can ignore until later?

Reply:
I'm pushing back because in the past, warning alerts fired and were visible to primary, but we felt that primary was overloaded by the existing set of critical alerts. If primary is expected to look at something it should be critical.

We could have a warning alert out there to collect data around the impact of this alert getting promoted to a critical one for example. Otherwise, I think this kind of "more important warning" alert should go through wider discussion among SREs. E.g. if I'm busy can I silence it? Does it count against the 3.5 alerts/hour SLO?